src/API/QueryFieldService.php

```diff
@@ -181,4 +180,20 @@ private function loadFieldDefinition(Content $content, string $fieldDefinitionId
 
         return $fieldDefinition;
     }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\Content\Query $query
+     *
+     * @return array
```

Comment:
You can skip those as they don't provide any extra information with respect to the method signature.